### ISSUE TITLE
DO it work with Xiaomi xiaofang iSC5 ?

### ISSUE THREAD

**Issue by greee-fr** [samtap/fang-hacks#323]
Xiaomi xiaofang Smart 1080P WiFi IP Camera (iSC5 model) can be view here
https://www.gearbest.com/ip-cameras/pp_620813.html
It is very cheap and sounds to have good quality (and 2 can be paired for a larger view!)
Does this hack work with this model ?

**Comment by CaffeinatedTech** [comment]
Yeah I just configured one.  Don't update the firmware in the Mi Home app.

**Comment by bend** [comment]
Hi, 
I have two Xiaomi xiaofang Smart 1080P WiFi IP Camera, and I tried to apply the hack, but I get a 404 page for the status page. When I go to hello.cgi I can see the snx_autorun.sh script. Any idea what i can do ? The firmware seems to be to the latest one in the mi app.

Thanks

**Comment by CaffeinatedTech** [comment]
You must downgrade the firmware.  Check out [this](https://github.com/samtap/fang-hacks/wiki/HowTo:-Flash-original-Xiaomi-firmware-from-sdcard-(factory-reset)#via-sdcard) guide.

**Comment by marmotte73** [comment]
I have an iSC5 model. On MiHome i have the firmware 5.6.2.138. I try to downgrade but it seems to be impossible. Have you got an idea ?

**Comment by watchaaa** [comment]
> I have an iSC5 model. On MiHome i have the firmware 5.6.2.138. I try to downgrade but it seems to be impossible. Have you got an idea ?

me too.
I have this model and firmware 5.6.2.43 and can't do anything, no downgrade and no cgi-bin status...
The only think that I can do is activate RTSP stream.

I want it for Domoticz and jpg image but not possible for me.... 
Very disappointed

**Comment by ghost** [comment]
> The only think that I can do is activate RTSP stream.

How?

**Comment by watchaaa** [comment]
> > The only think that I can do is activate RTSP stream.
> 
> How?

https://github.com/EliasKotlyar/Xiaomi-Dafang-Hacks/issues/718
